app/layout.tsx

```diff
@@ -28,13 +27,10 @@ export default function RootLayout({
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
-        <Provider>
-          {/* <Header /> */}
-          <div className="flex">
-            <div className="w-1/5">{/* <ChatList /> */}</div>
-            <div className="w-4/5">{children}</div>
-          </div>
-        </Provider>
+        <div className="flex">
+          <div className="w-1/5">{/* <ChatList /> */}</div>
```

Comment:
why commented?

Reply:
The component was deleted so maybe we should also delete this line